commands/push.go

```diff
@@ -43,4 +70,9 @@ var pushCmd = &cobra.Command{
 
 func init() {
 	RootCmd.AddCommand(pushCmd)
+
+	pushCmd.Flags().SortFlags = false
+
+	pushCmd.Flags().BoolVarP(&defaultRemote, "default", "d", false,
+		"If a remote is provided, set the default remote for future pushes. Otherwise list the default remote if one exists.")
```

Comment:
In term of UX that's not good. The `push` command should not have a lasting effect on the config. All the git-bug config manipulation should be centralized in a new `config` command. I guess it would be better to open an issue, see what it should cover and design this new command.

Reply:
Hmm, I see the argument, but a standard `git push` can already have a lasting effect on the repo config with e.g. `--set-upstream`. Even if there is a config command, it still seems like you should be able to both push to a remote and set that remote as the default at the same time.